sdcm/nemesis.py

```diff
@@ -5590,6 +5631,15 @@ def disrupt(self):
         self.disrupt_run_unique_sequence()
 
 
+class NemesisDeleteRangeSequence(Nemesis):
```

Comment:
Need to generate the nemesis list and the pipeline for this nemesis

Reply:
I will do it once when it will be ready and implementation and nemesis name will be approved